### PR TITLE
Bump Maven to 3.8.2

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
@ryanjbaxter @spencergibb please review

File: mvnw [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (spencergibb) [inline mvnw:11]: Has to be https

File: mvnw [hunk 2]
```diff
@@ -216,38 +194,117 @@ concat_lines() {
...
+fi
+
 # Provide a "standardized" way to retrieve the CLI args that will
 # work with both Windows and non-Windows executions.
 MAVEN_CMD_LINE_ARGS="$MAVEN_CONFIG $@"
 export MAVEN_CMD_LINE_ARGS
 
 WRAPPER_LAUNCHER=org.apache.maven.wrapper.MavenWrapperMain
 
-echo "Running version check"
```
Comment (spencergibb) [inline mvnw:229]: Removing this customization is bad as is is required

File: mvnw.cmd [hunk 3]
```diff
@@ -7,7 +7,7 @@
 @REM "License"); you may not use this file except in compliance
 @REM with the License.  You may obtain a copy of the License at
 @REM
-@REM    https://www.apache.org/licenses/LICENSE-2.0
+@REM    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (spencergibb) [inline mvnw.cmd:10]: Https again

**Review by spencergibb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**